src/components/parameters/ExtraParameters.svelte

```diff
@@ -0,0 +1,151 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import WarningExtraIcon from '@nasa-jpl/stellar/icons/warning_extra.svg?component';
+  import CopyIcon from 'bootstrap-icons/icons/copy.svg?component';
+  import { createEventDispatcher } from 'svelte';
+  import type { ArgumentsMap } from '../../types/parameter';
+  import { isMacOs } from '../../utilities/generic';
+  import { isMetaOrCtrlPressed } from '../../utilities/keyboardEvents';
+  import { permissionHandler } from '../../utilities/permissionHandler';
+  import { pluralize } from '../../utilities/text';
+  import { showFailureToast, showSuccessToast } from '../../utilities/toast';
+  import { tooltip } from '../../utilities/tooltip';
+  import Input from '../form/Input.svelte';
+
+  export let argumentsMap: ArgumentsMap;
+  export let hasPermission: boolean = true;
+  export let extraArguments: string[];
+  export let permissionError: string | undefined = undefined;
+
+  const dispatch = createEventDispatcher();
+
+  async function onCopy(event: MouseEvent) {
+    const { value } = event.currentTarget as HTMLButtonElement;
+    if (value) {
+      try {
+        await navigator.clipboard.writeText(`${value}`);
+        showSuccessToast('Value of parameter copied to clipboard');
+      } catch {
+        showFailureToast('Error copying parameter value to clipboard');
+      }
+    }
+  }
+
+  async function onReset(event: MouseEvent) {
+    if (isMetaOrCtrlPressed(event)) {
+      dispatch('reset', 'extra');
+    }
+  }
+</script>
+
+<div class="extra-parameters-container">
+  <fieldset>
+    <span class="extra-parameters-header"
+      >Extra Parameters
+      <button
+        class="reset-button"
+        value="extra"
+        on:click={onReset}
+        use:tooltip={{
+          allowHTML: true,
+          content: `<div class="activity-error-rollup-error">
+            <div class="activity-error-rollup-error-reset">
+                <span>Reset ${extraArguments.length} extra parameter${pluralize(
```

Comment:
Is the effect of this that these extra args will be deleted? If so I'm wondering if language like "Remove Extraneous Parameters" would make more sense? Also wondering if we should stick to "Extraneous" vs "Extra" since the former (to me at least) indicates that the parameters do not belong whereas the latter makes me think that it's ok to add extra parameters?